Source/Classes/NantesLabel.swift

```diff
@@ -181,6 +184,11 @@ public extension NSAttributedString.Key {
     /// defaults to .center
     open var verticalAlignment: NantesLabel.VerticalAlignment = .center
 
+    /// Force the label to only respond to touchesEnded taps that occur on the truncation token
+    ///
+    /// This doesn't stop taps on any data elements (links, addresses and the like)
+    open var forceTapsOntoAttributionToken: Bool = false
```

Comment:
The plan is to turn this into an enum users can set when configuring the label to decide what kind of tap handling they want.

Reply:
you mean someting like
```
enum TapHandling {
    case wholeLabel
    case tokenOnly
    case none
}
```
?

Reply:
Something like that. We've also been talking about an option set instead internally. Either way, it'll allow the users to pick and choose what kinds of things they want to have turned on easily.